notebooks/CheckTransforms.ipynb

```diff
@@ -0,0 +1,670 @@
+{
```

Comment:
Let's create a separate `CheckTransformsDetection` and put it to `examples/notebooks`